mmdet/models/necks/fpg.py

```diff
@@ -0,0 +1,399 @@
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule, caffe2_xavier_init, constant_init
+
+from ..builder import NECKS
+
+
+class Transition(nn.Module):
+    """Base class for transition.
+
+    Args:
+        in_channels (int): Number of input channels.
+        out_channels (int): Number of output channels.
+    """
+
+    def __init__(self, in_channels, out_channels):
+        super().__init__()
+        self.in_channels = in_channels
+        self.out_channels = out_channels
+
+    def forward(x):
+        pass
+
+
+class UpInterpolationConv(Transition):
+    """A transition used for up-sampling.
+
+    Firstly up-sampling the input by interpolation and follow by a
```

Comment:
Up-sample the input by interpolation then refines the feature by a convolution layer.